### PR TITLE
refactor: rename to CustomAuth

### PR DESCRIPTION
**Pull request by michaellee8**

### CODE REVIEW

**Review by phuctm97 — APPROVED** [review]
lgtm

**Comment by rathishubham7** [comment]
Please check the following: 
1. Please update the documentation 
2. Why are there 2 folders, auth and directAuth ?
3. Verison bumps are incorrect.
4. Duplicate podspecs

**Comment by michaellee8** [comment]
@rathishubham7 made changes, pls review again.

File: README.md [hunk 1]
```diff
@@ -17,16 +17,16 @@ In project settings, add the Github URL as a swift package dependency.
 import PackageDescription
 
 let package = Package(
-    name: "TorusSwiftDirectSDK", 
+    name: "CustomAuth", 
     dependencies: [
-        .package(name: "TorusSwiftDirectSDK", url: "https://github.com/torusresearch/torus-direct-swift-sdk", from: "1.0.0"))
+        .package(name: "CustomAuth", url: "https://github.com/torusresearch/custom-auth-swift-sdk", from: "2.0.0"))
```
Comment (rathishubham7) [inline README.md:22]: fix this link

File: cocoapods/Podfile.lock [hunk 2]
```diff
@@ -1,13 +1,13 @@
 PODS:
   - BigInt (5.0.0)
   - CryptoSwift (1.4.2)
+  - CustomAuth (2.2.0):
```
Comment (rathishubham7) [inline cocoapods/Podfile.lock:4]: fix podlock

**Pull request merged**